server.js

```diff
@@ -55,7 +55,14 @@
     app.use(function(req, res, next) {
         res.header('Access-Control-Allow-Origin', '*');
         res.header('Access-Control-Allow-Headers', 'Origin, X-Requested-With, Content-Type, Accept');
-        next();
+        //intercepts OPTIONS method
+        if ('OPTIONS' === req.method) {
```

Comment:
Is this still needed if we're just going to mock OPTIONS in the test? (If not I would remove it since people unfortunately use our code as a basis for their own server).